src/main/java/cart/common/config/RefundPolicyConfig.java

```diff
@@ -0,0 +1,19 @@
+package cart.common.config;
+
+import cart.domain.refund.RefundPolicy;
+import cart.domain.refund.RefundPolicyComposite;
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RefundPolicyConfig {
+
+    @Bean
+    public RefundPolicyComposite refundPolicyComposite(
+        final RefundPolicy fullRefundPolicy, final RefundPolicy halfRefundPolicy) {
+        return new RefundPolicyComposite(
+            List.of(fullRefundPolicy, halfRefundPolicy)
+        );
+    }
```

Comment:
다른 Policy 가 생기면 매번 Config 코드를 건드려줘야 겠군요 🤔 

List<RefundPolicy> 타입으로 받아보면 어떤 일이 일어나는지 학습 테스트 해보실까요?

```suggestion
    public RefundPolicyComposite refundPolicyComposite(
        final List<RefundPolicy> refundPolicies) {
        return new RefundPolicyComposite(refundPolicies);
    }
```

Reply:
<img width="344" alt="스크린샷 2023-06-08 오후 11 10 17" src="https://github.com/woowacourse/jwp-shopping-order/assets/46810837/93c8d77a-f5bd-40fd-af67-3dc0c1f87687">

디버깅을 통해서 확인해봤는데 List로도 받을 수 있네요!
컴포넌트 스캔 순서대로 등록이 되어서 사전순으로 Full -> Half Refund Policy가 등록되는 것을 확인했습니다!

다만, 혹시라도 순서가 이상하게 걸릴 수도 있으니까 아래와 같이 정책이 적용 가능한지 판단하는 메서드를 조금 더 상세하게 조건을 주었습니다! (제가 빠뜨렸던 부분이네요 😅) 

- 아래는 50%만 할인이 가능할 경우 변경된 코드 부분입니다.

**Before**
```java
@Component
public class HalfRefundPolicy implements RefundPolicy {

    ...

    @Override
    public boolean isAvailable(final Order order, final LocalDateTime currentTime) {
        return currentTime.isBefore(order.getOrderedAt().plusDays(HALF_REFUND.getDay()));
    }

```

**After**
```java
@Component
public class HalfRefundPolicy implements RefundPolicy {

    ...

    @Override
    public boolean isAvailable(final Order order, final LocalDateTime currentTime) {
        final LocalDateTime orderedAt = order.getOrderedAt();
        return currentTime.isAfter(orderedAt.plusDays(FULL_REFUND.getDay())) &&
            currentTime.isBefore(orderedAt.plusDays(HALF_REFUND.getDay()));
    }
```